cleanrl_utils/huggingface.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+import os
+import sys
+from pathlib import Path
+from pprint import pformat
+from typing import List
+
+import numpy as np
+
+HUGGINGFACE_VIDEO_PREVIEW_FILE_NAME = "replay.mp4"
+HUGGINGFACE_README_FILE_NAME = "README.md"
+
+
+def push_to_hub(
+    args: argparse.Namespace,
+    episodic_returns: List,
+    repo_id: str,
+    algo_name: str,
+    folder_path: str,
+    video_folder_path: str = "",
+):
```

Comment:
(suggestion)

If you have a single `create_commit(...)` call, it can be easy to add two parameters:
- `revision: Optional[str] = None`: allows to push to a different branch on the repo. Push on `main` by default.
- `create_pr: bool = False`: allows to create a PR in case user is not pushing on its own repo.
Both params can be provided to `create_commit` without any other processing on `cleanrl` side.

Also possible to add a `private: bool = False` argument that can be passed to `create_repo(...)` to create a private repo instead.

Reply:
Done.